### DISCUSSION TITLE
some of the textures are missing

### DISCUSSION THREAD

**Discussion by cats2333** [bevyengine/bevy#9887] [Q&A]
Excuse me,Why is this tank model correct in Blender, and the UV is also correct, but after importing Bevy, it appears that some of the textures are missing, and the turret only has one material

**Comment by cats2333** [comment]
![1](https://github.com/bevyengine/bevy/assets/19430019/c15f6583-cc41-4b42-bb99-f24e389a7b9a)
![2](https://github.com/bevyengine/bevy/assets/19430019/f8df27dd-58e8-4e4e-95d2-748ad263d39e)

**Comment by superdump** [comment]
Could be winding order of the triangles being wrong so the triangles are being back-face culled. You could set double-sided and face culling to None on the material to test that. Could be that the normals are facing the wrong direction and that's perhaps more tricky to test.

**Reply by cats2333** [reply]
Okay, thanks a lot!
